addon/components/paper-sidenav-toggle.js

```diff
@@ -1,27 +1,26 @@
 import Ember from 'ember';
-import PaperNavContainer from './paper-nav-container';
+const { Component } = Ember;
 
-export default Ember.Component.extend({
-  tagName: 'div',
+export default Component.extend({
 
-  navContainer: Ember.computed(function() {
-    return this.nearestOfType(PaperNavContainer);
-  }),
+  classNameBindings: ['hideClass'],
 
-  actions: {
-    toggleMenu() {
-      this.get('navContainer').get('sideBar').send('toggleMenu');
-    }
-  },
+  name: 'default',
 
-  didInsertElement() {
+  init() {
     this._super(...arguments);
     if (this.get('navContainer')) {
-      let lockedOpen = this.get('navContainer').get('sideBar').get('locked-open');
+      let lockedOpen = this.get('navContainer').get('sideBar').get('lockedOpen');
```

Comment:
Would there be a use case for lockedOpen being dynamic (i.e. changing after component creation)? I don't have one, but it would be easy to move this to didReceiveAttrs().